packages/waku/src/router/create-pages.ts

```diff
@@ -107,19 +108,19 @@ export type CreatePage = <
   WildSlugKey extends string,
   Render extends 'static' | 'dynamic',
   StaticPaths extends StaticSlugRoutePaths<Path>,
+  EP extends boolean | undefined = undefined,
```

Comment:
Can we make it explicit?
```suggestion
  ExactPath extends boolean | undefined = undefined,
```